root/defaults/autostart

```diff
@@ -1,22 +1,2 @@
 #! /bin/bash
```

Comment:
```suggestion
#!/bin/bash
```
Definitely shouldn't use a space here, but also, should this use `with-contenv` ?

Reply:
Follow up - i did some extra digging and found the space in the shebang is perfectly fine. most of our other scripts don't have it, so it's just a code style consistency preference. So mainly just the `with-contenv` concern here since this uses `CHROME_CLI` from the container env.

Reply:
bash header is intentional in all base gui containers as with-contenv is not available when using /kasminit https://github.com/linuxserver/docker-baseimage-kasmvnc/blob/master/root/kasminit, but the users env is already injected at this point with https://github.com/linuxserver/docker-baseimage-kasmvnc/blob/master/root/etc/s6-overlay/s6-rc.d/svc-de/run. This allows the container to run in usermode. 
The header is technically not even needed at this point unless the user or us goes off posix and uses bash only logic as this is launched with sh by openbox: 
https://github.com/danakj/openbox/blob/9e8813e111cbe6c1088f6abbc771a29470f05fc2/data/autostart/openbox-autostart.in#L26